### DISCUSSION TITLE
blog/daily/11%20-%20My%20Year%20of%202023

### DISCUSSION THREAD

**Discussion by giscus** [earayu/earayu.github.io#2] [General]
# blog/daily/11%20-%20My%20Year%20of%202023

Hi, I'm earayu. This year, I resigned from a job where I worked for over five years and left the city where I lived for more than six years to return to my hometown, Suzhou. I began working from home and engaging in open-source projects. This has been my 2023.

http://localhost:3000/blog/daily/11%20-%20My%20Year%20of%202023

**Comment by earayu** [comment]
test comment

**Comment by qrkks** [comment]
很棒的生活
